apps/api/pages/api/bookings/_get.ts

```diff
@@ -162,6 +162,9 @@ function buildWhereClause(
 
 async function handler(req: NextApiRequest) {
   const { userId, isAdmin, prisma } = req;
+
+  const { dateFrom, dateTo } = schemaBookingGetParams.parse(req.query);
```

Comment:
@alishaz-polymath can you take a look at this?

Reply:
<img width="574" alt="Screenshot 2024-01-08 at 3 35 00 PM" src="https://github.com/calcom/cal.com/assets/36115192/490c1fbf-214f-4449-8e86-08bbd682f303">